src/js/index.js

```diff
@@ -2,7 +2,7 @@ import load from "./load";
 
 var profileId = 1;
 var baseUrl = "https://wazimap-ng.openup.org.za";
-if (window.location.href.search("localhost") > 0)
-    baseUrl = "http://localhost:8001";
+// if (window.location.href.search("localhost") > 0)
```

Comment:
Don't commit this change, it's only a workaround since you aren't hosting the backend on your machine